audit/projects/kubernetes-public/secrets/slackin-token/versions.json

```diff
@@ -1,11 +1,20 @@
 [
+  {
+    "createTime": "2021-09-27T15:30:16.543300Z",
+    "etag": "\"15ccfbc6223c44\"",
+    "name": "projects/127754664067/secrets/slackin-token/versions/2",
+    "replicationStatus": {
+      "automatic": {}
+    },
+    "state": "ENABLED"
+  },
```

Comment:
Basically all the slack-infra secrets stuff is https://github.com/kubernetes/k8s.io/pull/2828

Either creating the new secrets, or populating new versions for each secret to support syncing from kubernetes-external-secrets